server/data/manageUsersApiClient.ts

```diff
@@ -0,0 +1,38 @@
+import type TokenStore from './tokenStore'
+import logger from '../../logger'
+import config from '../config'
+import RestClient from './restClient'
+
+export interface User {
+  username: string
+  name?: string
+  active?: boolean
+  authSource?: string
+  uuid?: string
+  userId?: string
+  staffId?: number // deprecated, use userId
+  activeCaseLoadId?: string // deprecated, use user roles api
+}
+
+export interface UserRole {
+  roleCode: string
+}
+
+export default class ManageUsersApiClient {
+  constructor(private readonly tokenStore: TokenStore) {}
```

Comment:
Don't need a token store, this is only an auth specific thing, never used in the methods below